__tests__/integration/utils/test-utils-integration.js

```diff
@@ -157,7 +177,11 @@ export class TestUtils {
       }
     }
     // Log the success and return
-    loggers.test.informNewWallet(walletObj.walletId, walletObj.words);
+    if (walletObj.words) {
+      loggers.test.informNewWallet(walletObj.walletId, walletObj.words);
+    } else {
+      loggers.test.informNewWallet(walletObj.walletId, `seedKey: ${walletObj.seedKey}`);
```

Comment:
Should we log if it's a multisig?

Reply:
Done in [ba2463d](https://github.com/HathorNetwork/hathor-wallet-headless/pull/156/commits/ba2463dfde5698885fdaa492d44c496987bcfd1d)